iommi/style_base.py

```diff
@@ -79,6 +85,8 @@
     ),
     Query=dict(
         template='iommi/query/form.html',
+        assets__ajax_enhance__template='iommi/table/ajax_enhance.html',
```

Comment:
Would it not be more natural to have it at `iommi/query/ajax_enhance.html`

Reply:
Or on `Table`...

Reply:
But it is only needed when we filter, right? When just using Table on a list of objetcs this is not a thing.

Reply:
True. And I guess we could tweak it so it can work on a non-table Query target too in the future when the need arrives.